core/includes/stream_wrappers.inc

```diff
@@ -147,8 +147,8 @@ interface BackdropStreamWrapperInterface extends StreamWrapperInterface {
    *
    * This function should return a URL that can be embedded in a web page
    * and accessed from a browser. For example, the external URL of
-   * "youtube://xIpLd0WQKCY" might be
-   * "http://www.youtube.com/watch?v=xIpLd0WQKCY".
+   * "youtube://123aBc456xYz" might be
+   * "http://www.youtube.com/watch?v=123aBc456xYz".
```

Comment:
this video doesn't exist, but I prefer that to what's there now :)

Reply:
This was tripping up cspell, so I wanted to make it happy but without having to add `xIpLd0WQKCY` to the list of known words in the dictionary. We could wrap this line or just the URL in a `cspell:disable`/`cspell:enable` pair of keywords, but then those keywords would end up in the function description in the API site. Or we could wrap the entire docblock in the cspell keywords, which then has the risk of introduction of typos for the entire docblock in the future.